widgets/shortvideo/_inline-tile.scss

```diff
@@ -0,0 +1,48 @@
+@use "sass:map";
+@use "@styles/partials/media-queries";
+@use "dimens";
+
+.shortvideo-inline {
+  .ugc-tile {
+    position: relative;
+    width: 100%;
+    display: flex;
+
+    .tile {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+
+      .tile-image-wrapper {
+        display: flex;
+        width: 100%;
+        height: var(--tile-size) !important;
+
+        img {
+          aspect-ratio: auto;
+          width: 100%;
+          object-fit: cover;
+          overflow: hidden;
+          border-radius: dimens.$inline-tile-border-radius;
+          position: relative;
+          opacity: 1;
+        }
+      }
+    }
+  }
+
+  .icon-play {
+    width: dimens.$play-icon_width;
+    height: dimens.$play-icon_height;
+    background-image: url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 30 30" fill="none"><circle cx="15" cy="15" r="15" fill="%23BCBBBC"/><path d="M19.5 14.1336C20.1667 14.5185 20.1667 15.4808 19.5 15.8657L13.5 19.3298C12.8333 19.7147 12 19.2335 12 18.4637L12 11.5355C12 10.7657 12.8333 10.2846 13.5 10.6695L19.5 14.1336Z" fill="black"/></svg>');
+    background-size: contain;
+  }
+
+  & ~ .swiper-button-next,
+  & ~ .swiper-button-prev {
+    --swiper-navigation-sides-offset: 0px;
+
+    top: calc(var(--swiper-navigation-top-offset, 50%) - dimens.$nosto-container-margin + 10) !important;
```

Comment:
Is the `!important` flag necessary?

Reply:
This is not needed, thanks for pick this up